pkgs/tools/text/ugrep/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, stdenv, pcre2, boost, zlib, lzham }:
+
+stdenv.mkDerivation rec {
+  pname = "ugrep";
+  version = "2.4.1";
+
+  src = fetchFromGitHub {
+    owner = "Genivia";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "16nr7zq4l2si9pfckfinbqnv94hw51z2qcbygc9x81jbjlvg3003";
+  };
+
+  buildInputs = [ pcre2 boost zlib lzham ];
+
+  meta = with stdenv.lib; {
```

Comment:
`with <attrset>` is apparently [another anti-pattern](https://nix.dev/anti-patterns/language.html#with-attrset-expression).